coalib/parsing/StringProcessing/Core.py

```diff
@@ -328,6 +328,38 @@ def escape(string, escape_chars, escape_with="\\"):
     return string
 
 
+def convert_to_raw(string, exceptions=""):
+    """
+    Converts a string to its raw form, converting all backslash to double
+    backslash except when the backslash escapes a character given in
+    exceptions.
+
+    :param string:     The given string that needs to be converted
+    :param exceptions: A list of characters that if escaped with backslash
+                       should not be converted to double backslash.
+    :return:           Returns the corresponding raw string.
+    """
+    exceptions += "/"
```

Comment:
I still don't see why we need to escape the slash.

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-04-11 13:08 GMT+02:00 Adhityaa Chandrasekar <notifications@github.com>:

> In coalib/parsing/StringProcessing/Core.py
> <https://github.com/coala-analyzer/coala/pull/1959#discussion_r59187429>:
>
> > @@ -328,6 +328,38 @@ def escape(string, escape_chars, escape_with="\\"):
> >      return string
> >
> >
> > +def convert_to_raw(string, exceptions=""):
> > +    """
> > +    Converts a string to its raw form, converting all backslash to double
> > +    backslash except when the backslash escapes a character given in
> > +    exceptions.
> > +
> > +    :param string:     The given string that needs to be converted
> > +    :param exceptions: A list of characters that if escaped with backslash
> > +                       should not be converted to double backslash.
> > +    :return:           Returns the corresponding raw string.
> > +    """
> > +    exceptions += "/"
>
> I'll add a comment saying the same.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1959/files/16f3a8eaa4025d97178405c33fc146c70e9d998d..778844908cc8bd0428c4788790421dda53ea6cc6#r59187429>
>